File: python/1 star/grade.py
score = int(input())
if score >= 0 and score <= 100 :
  if score >= 80 and score <= 100 :
    print("A")
  if score >= 70 and score <= 79 :
    print("B")
  if score >= 60 and score <= 69 :
    print("C")
  if score >= 50 and score <= 59 :
    print("D")
  if score < 50 :
    print("F")
else:print("ERROR")

File: javascript/1 star/grade.js
const rl = require("readline").createInterface(process.stdin);
let Input = 0

rl.on("line", (line) => {
	Input = line
}).on("close", () => {
  if (Input >= 80) console.log("A")
  if (Input >= 70 && Input < 80) console.log("B")
  if (Input >= 60 && Input < 70) console.log("C")
  if (Input >= 50 && Input < 60) console.log("D")
  if (Input < 50) console.log("F")
});
